src/main/steps/common/form/form.njk

```diff
@@ -105,7 +105,7 @@
             attributes: {
               id: 'main-form-cancel'
             }
-          }) if getContent(form.cancel.text)
+          }) if getContent(form.goBack.text)
```

Comment:
this change is in common form.njk. justify this change?

Reply:
followed the same as for other buttons like submit, cancel